include/envoy/common/platform.h

```diff
@@ -227,6 +231,9 @@ typedef int os_fd_t;
 #define SOCKET_ERROR_INVAL EINVAL
 #define SOCKET_ERROR_ADDR_IN_USE EADDRINUSE
 
+namespace Platform {
+constexpr std::string_view null_device_path{"/dev/null"};
```

Comment:
I thought we had a linter on this, but please use absl::string_view for now, as I think there are issues with C++17 string view on some platforms.

Reply:
thanks did not know about this. It is fixed in 1afe60a360e801678b7edca96b07ea4ce2158edd